controllers/core/openstackversion_controller.go

```diff
@@ -301,6 +301,15 @@ func (r *OpenStackVersionReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		Log.Info("Setting DeployedVersion")
 		instance.Status.DeployedVersion = &instance.Spec.TargetVersion
 	}
+	if instance.Status.AvailableVersion != nil &&
+		instance.Status.DeployedVersion != nil &&
+		*instance.Status.AvailableVersion != *instance.Status.DeployedVersion {
+		instance.Status.Conditions.Set(condition.TrueCondition(
+			corev1beta1.OpenStackVersionMinorUpdateAvailable,
+			corev1beta1.OpenStackVersionMinorUpdateAvailableMessage))
```

Comment:
Don't we need to Init this new condition up top? Also could we add a functional test to verify this?

Reply:
the condition is only set when there is an update available. if not, we remove it in the else. so don't think we have to init it.

Reply:
will add test